GlotPress/GlotPress-WP
# [1813] Fix errors when URL parameters like "filter[status]" aren't strings

## Problem

If the translations filter in the URL parameter sends an array, for example `...?filters[status][0]=current&filters[status][1]=fuzzy`, the `gp_array_get( $filters, 'status', 'current_or_waiting_or_fuzzy_or_untranslated' )` will return an array instead of a string.
This errors on subsequent `explode( '_or_', $selected_status )`.

## Solution

If the statuses set on URL get parameters are not the predictable ones, fallback to ~the `current_or_waiting_or_fuzzy_or_untranslated`~ **an empty** string.

## Testing Instructions
1. Open a translation set.
2. Change the default `...?filters[status]=current_or_fuzzy` URL parameters to `...?filters[status][0]=current&filters[status][1]=fuzzy`
3. See the errors. 

Fixes #1813
